src/main/java/com/ceos22/cgvclone/domain/reservation/controller/ReservationController.java

```diff
@@ -0,0 +1,32 @@
+package com.ceos22.cgvclone.domain.reservation.controller;
+
+import com.ceos22.cgvclone.domain.reservation.dto.ReservationCancelDTO;
+import com.ceos22.cgvclone.domain.reservation.dto.ReservationRequestDTO;
+import com.ceos22.cgvclone.domain.reservation.dto.ReservationResponseDTO;
+import com.ceos22.cgvclone.domain.reservation.service.ReservationService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequiredArgsConstructor
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    @PostMapping("/api/reservation")
+    public ResponseEntity<ReservationResponseDTO> createReservation(@RequestBody ReservationRequestDTO reservationRequestDTO,
+                                                                    @AuthenticationPrincipal org.springframework.security.core.userdetails.User user) {
```

Comment:
p4.
```suggestion
                                                                    @AuthenticationPrincipal User user) {
```
여기는 User Entity를 안쓰고 있어서 생략이 가능해보이네요.
단, 이러한 이슈가 계속 발생하고 클래스 명이 같아 가독성도 떨어질 것 같아요

Reply:
알려주셔서 감사합니다. 정돈해보겠습니다!